test/framework/integration_framework/fake_peer/fake_peer.hpp

```diff
@@ -98,11 +109,15 @@ namespace integration_framework::fake_peer {
     ProposalStorage &getProposalStorage();
 
     /// Start the fake peer.
-    std::unique_ptr<iroha::network::ServerRunner> run();
+    std::unique_ptr<iroha::network::ServerRunner> run(bool reuse_port = false);
```

Comment:
I would add comment explaining reasoning of new parameter. Probably worth mentioning that was added to run tests in parallel?

Reply:
commented